packages/react/src/button/styles.ts

```diff
@@ -102,9 +104,11 @@ export function buttonStyles({
 	block: boolean;
 	size: ButtonSize;
 	variant: ButtonVariant;
+	alignSelf?: CSSObject['alignSelf'];
```

Comment:
I don't think we should use `CSSObject` because we only support a subset of the available values

Reply:
Yeah, fair enough. I couldn't figure out how to make this skip the responsive stuff. I'll try again.

- [ ] No CSSObject

Reply:
Interesting. Why not add `mq` to make buttons responsive? Feels weird to have hindered implementation of `alignSelf`. I could see buttons changing from `block` to not at breakpoints too

Reply:
Hmm yeah, that's true, `stretch` on mobile and `start` thereafter sounds a common use. Ok, thanks I'll do that.